test/e2e/tests/dapp-interactions.spec.js

```diff
@@ -34,8 +31,7 @@ describe('Dapp interactions', function () {
         await driver.fill('#password', 'correct horse battery staple');
         await driver.press('#password', driver.Key.ENTER);
 
-        // Connect to Dapp0
-        await connectDappWithExtensionPopup(driver, 0);
```

Comment:
We do not actually need to connect to the dapp to click the add ethereum chain button